frontend/src/utils/Tabs.svelte

```diff
@@ -0,0 +1,65 @@
+<style>
+  button {
+    margin-bottom: 0;
+    border: 1px solid white;
+  }
+
+  .tabs button:focus {
+    border-bottom: 2px solid var(--main-bg-color);
+    outline: 0;
+  }
+
+  .tabs button {
+    margin-right: 0.5em;
+    border-bottom: 0px;
+  }
+
+  hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid white;
+    margin-top: -1px;
+    padding: 0;
+  }
+</style>
+
+<script>
+  import { onMount } from "svelte";
+  export let tabs, tabId, defaultTab;
+  onMount(async () => {
```

Comment:
```suggestion
  onMount(() => {
```

This doesn't need to be `async`, right?